### PR TITLE
Create mermaid diagram for Substrate Node

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR creates a mermaid diagram for Substrate Node architecture. It demonstrates the separation between the runtime and the client. 

@nhussein11 I got this laid out ok but I'm not having luck applying styles. I was hoping to make the "Substrate Node" text a size bigger and maybe bold? idk. Just doesn't stand out much. Do you want to take a shot at applying styles?

### CODE REVIEW

File: develop/parachains/intro-polkadot-sdk.md [hunk 1]
```diff
@@ -50,6 +50,21 @@ Every blockchain platform relies on a decentralized network of computers—calle
...
+    subgraph sg1[Substrate Node]
+        B[Wasm Runtime -</br>application logic]
+        I[RuntimeCall Executor]
+        subgraph sg2[Client]
+            direction TB
+            C[Network and blockchain</br>infrastructure activity]
+        end
+        I -.-> B
+    end
+
```
Comment (0xLucca) [inline develop/parachains/intro-polkadot-sdk.md:65]:
```suggestion
graph TB
    subgraph sg1[Substrate Node]
        B[Wasm Runtime -</br>application logic]
        I[RuntimeCall Executor]
        subgraph sg2[Client]
            direction TB
            C[Network and blockchain</br>infrastructure activity]
        end
        I -.-> B
    end
```

File: develop/parachains/intro-polkadot-sdk.md [hunk 2]
```diff
@@ -50,6 +50,21 @@ Every blockchain platform relies on a decentralized network of computers—calle
     - Stored as a part of the chain state
     - Also known as State Transition Function (STF)
 
+``` mermaid
+
+graph TB
+    subgraph sg1[Substrate Node]
+        B[Wasm Runtime -</br>application logic]
```
Comment (0xLucca) [inline develop/parachains/intro-polkadot-sdk.md:57]:
```suggestion
        B[Wasm Runtime -</br>STF]
```

File: develop/parachains/intro-polkadot-sdk.md [hunk 3]
```diff
@@ -50,6 +50,21 @@ Every blockchain platform relies on a decentralized network of computers—calle
...
+        I[RuntimeCall Executor]
+        subgraph sg2[Client]
+            direction TB
+            C[Network and blockchain</br>infrastructure activity]
```
Comment (0xLucca) [inline develop/parachains/intro-polkadot-sdk.md:61]: This description feels weird. The word `activity` does not sound good to me. Maybe replacing it with `services` makes it work

**Comment by nhussein11** [comment]
> @nhussein11 I got this laid out ok but I'm not having luck applying styles. I was hoping to make the "Substrate Node" text a size bigger and maybe bold? idk. Just doesn't stand out much. Do you want to take a shot at applying styles?

@dawnkelly09 Definitely! I can take a shot at the styles once Luca's comments are addressed :)

File: develop/parachains/intro-polkadot-sdk.md (hunk 3 above)
Comment (dawnkelly09) [reply develop/parachains/intro-polkadot-sdk.md:61]: Cool, cool! I just pulled this off substrate docs. Will update!

**Review by nhussein11 — APPROVED** [review]

**Review by 0xLucca — APPROVED** [review]
LGTM!

**Pull request merged**